VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -234,8 +236,13 @@ public void onClick(View view) {
             public void onClick(View v) {
                 String to = environment.getConfig().getFeedbackEmailAddress();
                 String subject = getString(R.string.email_subject);
-                String email = "";
-                EmailUtil.sendEmail(getActivity(), to, subject, email, environment.getConfig());
+
+                String versionName = PropertyUtil.getManifestVersionName(getActivity());
+                String osVersionText = String.format("%s %s", getString(R.string.android_os_version), android.os.Build.VERSION.RELEASE);
+                String appVersionText = String.format("%s %s", getString(R.string.app_version), BuildConfig.VERSION_NAME);
+                String deviceModelText = String.format("%s %s", getString(R.string.android_device_model), Build.MODEL);
+                String body = osVersionText + "\n" + appVersionText + "\n" + deviceModelText;
+                EmailUtil.openEmailClient(getActivity(), to, subject, body, environment.getConfig());
```

Comment:
Should we put several new lines at the beginning to leave room for their message? Or do we assume they will start typing *after* this data.